tools/generate_docker_image_names.py

```diff
@@ -0,0 +1,111 @@
+"""
+Dockerリポジトリ名、バージョン文字列、カンマ区切りのプレフィックスを受け取り、
+バージョン文字列付きのDockerイメージ名を改行区切りで標準出力に出力する
+
+例1
+$ python3 ./tools/generate_docker_image_names.py \
+  --repository "REPO" \
+  --version "VER" \
+  --prefix ",A,B"
+REPO:VER
+REPO:A-VER
+REPO:B-VER
+
+例2
+$ python3 ./tools/generate_docker_image_names.py \
+  --repository "REPO" \
+  --version "VER" \
+  --prefix ""
+REPO:VER
+"""
+
+from argparse import ArgumentParser
+
+
+def generate_docker_image_names(
+    repository: str, version: str, comma_separated_prefix: str
+) -> list[str]:
+    """
+    Dockerリポジトリ名、バージョン文字列、カンマ区切りのプレフィックスを受け取り、
+    バージョン文字列付きのDockerイメージ名を配列で返す
+
+    prefixが空文字列でない場合、"{prefix}-{バージョン文字列}"をタグにする
+    - 例: prefix="cpu", version="latest"
+           -> "voicevox/voicevox_engine:cpu-latest"
+
+    prefixが空文字列の場合、"{バージョン文字列}"をタグにする
+    - 例: prefix="", version="0.22.0" -> "voicevox/voicevox_engine:0.22.0"
+
+    Parameters
+    ----------
+    repository : str
+        Dockerリポジトリ名（例: "voicevox/voicevox_engine"）
+    version : str
+        バージョン文字列（例: "0.22.0", "latest"）
+    comma_separated_prefix : str
+        カンマ区切りのプレフィックス（例: ",cpu,cpu-ubuntu22.04", "nvidia,nvidia-ubuntu22.04"
+
+    Returns
+    -------
+    list[str]
+        Dockerイメージ名の配列
+        （例: [
+            "voicevox/voicevox_engine:0.22.0",
+            "voicevox/voicevox_engine:cpu-0.22.0",
+            "voicevox/voicevox_engine:cpu-ubuntu22.04-0.22.0"
+        ]）
+    """
```

Comment:
スクリプトの頭の説明文が汎用的な説明になったので、
関数の説明文は同じように汎用的な説明にして、
引数の説明文は具体例のままという形でコミットしてみます。

```suggestion
    """
    Dockerリポジトリ名、バージョン文字列、カンマ区切りのプレフィックスを受け取り、
    バージョン文字列付きのDockerイメージ名を配列で返す

    prefixが空文字列でない場合、"{prefix}-{version}"をタグにする

    - 例: repository="REPO", version="VER", prefix="A" -> "REPO:A-VER"

    prefixが空文字列の場合、"{version}"をタグにする

    - 例: repository="REPO", version="VER",  prefix="" -> "REPO:VER"

    Parameters
    ----------
    repository : str
        Dockerリポジトリ名（例: "voicevox/voicevox_engine"）
    version : str
        バージョン文字列（例: "0.22.0", "latest"）
    comma_separated_prefix : str
        カンマ区切りのプレフィックス（例: ",cpu,cpu-ubuntu22.04", "nvidia,nvidia-ubuntu22.04"

    Returns
    -------
    list[str]
        Dockerイメージ名の配列
        （例: [
            "voicevox/voicevox_engine:0.22.0",
            "voicevox/voicevox_engine:cpu-0.22.0",
            "voicevox/voicevox_engine:cpu-ubuntu22.04-0.22.0"
        ]）
    """
```